Dockerfile.s390x

```diff
@@ -133,7 +133,7 @@ RUN useradd --create-home --gid docker unprivilegeduser
 
 VOLUME /var/lib/docker
 WORKDIR /go/src/github.com/docker/docker
-ENV DOCKER_BUILDTAGS apparmor selinux
+ENV DOCKER_BUILDTAGS apparmor pkcs11 selinux
```

Comment:
I don't recall anything, and this works with both gccgo and power. Is pkcs11 used for anything other than notary, because you might not want to include it with gccgo seeing as we aren't building notary there.